docs/fastdds/discovery/static.rst

```diff
@@ -77,54 +77,88 @@ A full example of such file can be found in :ref:`static_xml_example`.
-| ``<unicastLocator>``   | Unicast locator of the DomainParticipant. |br|   |                        |                 |
-|                        | See :ref:`staticLocators`.                       |                        |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
-| ``<multicastLocator>`` | Multicast locator of the DomainParticipant. |br| |                        |                 |
-|                        | See :ref:`staticLocators`.                       |                        |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
-| ``<reliabilityQos>``   | See the :ref:`reliabilityqospolicy` section.     | |besteffort|           | |besteffort|    |
-|                        |                                                  +------------------------+                 |
-|                        |                                                  | |reliable|             |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
-| ``<durabilityQos>``    | See the :ref:`durabilityqospolicy` section.      | |volatile|             | |volatile|      |
-|                        |                                                  +------------------------+                 |
-|                        |                                                  | |transientlocal|       |                 |
-|                        |                                                  +------------------------+                 |
-|                        |                                                  | |transient|            |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
-| ``<ownershipQos>``     | See :ref:`ownershipQos`.                         |                        |                 |
-|                        |                                                  |                        |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
-| ``<livelinessQos>``    | Defines the liveliness of the remote peer. |br|  |                        |                 |
-|                        | See :ref:`livelinessQos`.                        |                        |                 |
-+------------------------+--------------------------------------------------+------------------------+-----------------+
+
+.. list-table::
+   :header-rows: 1
+   :align: left
+
+   * - Name
+     - Description
+     - Values
+     - Default
+   * - ``<userId>``
+     - Mandatory.
+       Uniquely identifies the DataReader/DataWriter.
```

Comment:
No new line is introduced this way. Maybe alias `|br|` is still required.

Reply:
Done

Reply:
There are still some more lines that were broken previously. Specially in the `Values` column I think it would be better to see each option in its own line and not one after the other in the same line.

Reply:
Done